signalfx/resource_signalfx_data_link.go

```diff
@@ -69,8 +69,7 @@ func dataLinkResource() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"is_default": &schema.Schema{
 							Type:        schema.TypeBool,
-							Default:     true,
-							Optional:    true,
+							Computed:    true,
```

Comment:
What computes it?

Reply:
in the getPayloadDataLink, I hardcoded it to false for both external link and target_splunk 
The idea is not to make this field configurable , but we still need to set it in the struct, so it should be defined in the schema... Is there a different option to use?